packages/node/src/evm/initialization.ts

```diff
@@ -1,4 +1,5 @@
 import { ethers } from 'ethers';
```

Comment:
This file is a bit of a mess now. I think the larger functions could be broken up into smaller functions

Reply:
Feel free to merge now or after you do this